engines/joomla/nucleus/particles/contentarray.html.twig

```diff
@@ -21,7 +21,12 @@
     {% set categories = joomla.finder('category', category_options).published(1).language().limit(0).find() %}
 
     {# Content Finder #}
-    {% set article_finder = joomla.finder('content').category(categories).published(1).language() %}
+    {% if filter.articles %}
+        {% set article_options = filter.articles ? {id: [filter.articles|split(',')]} : {} %}
```

Comment:
Hey Djamil,

Yeah, it's not a problem to apply the change you suggested.
But I already tested it (the current/original version in the PR) with all of the following and it works just fine:

```
1,2,5,6,10
```

```
1, 2,5,6 ,10
```

```
1,2,5,6,10, (notice the comma at the end)
```

```
1 , 2 , 5 , 6 , 10
```

Basically, I'm doing the same thing that Matias did with the "Categories":

```
{% set category_options = filter.categories ? {id: [filter.categories|split(','), 0]} : {} %}
```

So yeah, just let me know if you still want me to apply the change you suggested and I'll be happy to do it :wink: